install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -168,56 +303,92 @@ To enable writing to NFS volumes with SELinux enforcing on each node, run:
 # setsebool -P virt_use_nfs 1
 ----
 
-The `-P` option makes the bool persistent between reboots.
+The `-P` option above makes the bool persistent between reboots.
+
+The *virt_use_nfs* boolean is defined by the *_docker-selinux_* package. If an
+error is seen indicating that this bool is not defined, then ensure that this
+package has been installed.
+
+[[nfs-export-settings]]
+=== Export Settings
 
-Additionally, in order to enable arbitrary container users to read and write the
-volume, each exported volume on the NFS server itself should conform to the
-following:
+In order to enable arbitrary container users to read and write the volume, each
+exported volume on the NFS server should conform to the following:
```

Comment:
Suggest: "... following conditions." (w/ full stop).  This will make it easier to translate, i believe.